src/service/extension-location.js

```diff
@@ -34,12 +39,15 @@ function calculateScriptBaseUrl(location, isLocalDev) {
  * Calculate script url for an extension.
  * @param {!Location} location The window's location
  * @param {string} extensionId
+ * @param {string=} extensionVer
  * @param {boolean=} isLocalDev
  * @return {string}
  */
-export function calculateExtensionScriptUrl(location, extensionId, isLocalDev) {
+export function calculateExtensionScriptUrl(location, extensionId,
+    extensionVer = DefaultExtensionVersion, isLocalDev) {
   const base = calculateScriptBaseUrl(location, isLocalDev);
-  return `${base}/rtv/${getMode().rtvVersion}/v0/${extensionId}-0.1.js`;
+  return `${base}/rtv/${getMode().rtvVersion}/v0/`
+      + `${extensionId}-${extensionVer}.js`;
```

Comment:
I find `#calculateExtensionScriptUrl` is never called with `extensionVer=undefined`.  let's make it a required param, and only make it optional in `#loadExtension`
Also in this case, `DefaultExtensionVersion` should be moved to extensions-impl.js.